core/contract/src/main/java/org/eclipse/dataspaceconnector/contract/negotiation/ProviderContractNegotiationManagerImpl.java

```diff
@@ -484,6 +520,16 @@ public Builder dispatcherRegistry(RemoteMessageDispatcherRegistry dispatcherRegi
             manager.dispatcherRegistry = dispatcherRegistry;
             return this;
         }
+        
+        public Builder commandQueue(ContractNegotiationCommandQueue commandQueue) {
+            manager.commandQueue = commandQueue;
+            return this;
+        }
+        
+        public Builder commandRunner(ContractNegotiationCommandRunner commandRunner) {
+            manager.commandRunner = commandRunner;
```

Comment:
I don't think it can. The CommandRunner requires a CommandHandlerRegistry in its constructor. Since the managers do not have a reference to a CommandHandlerRegistry, we don't have one available in the builder. And even if we did, we could not be sure that the registry would have been set before the CommandQueue.

Reply:
oh, you're right, my oversight